tests/DevBetterWeb.Tests/Services/DailyCheckPingServiceTests/CheckIfAnyActiveInvitationsRequireAdminPing.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using DevBetterWeb.Core.Entities;
+using DevBetterWeb.Core.Interfaces;
+using DevBetterWeb.Infrastructure.Identity.Data;
+using DevBetterWeb.Infrastructure.Services;
+using Microsoft.AspNetCore.Identity;
+using Moq;
+using Xunit;
+
+namespace DevBetterWeb.Tests.Services.DailyCheckPingServiceTests
+{
+  public class CheckIfAnyActiveInvitationsRequireAdminPing
+  {
+    private readonly Mock<IRepository<Invitation>> _repository;
+    private readonly Mock<IEmailService> _emailService;
+    private readonly Mock<UserManager<ApplicationUser>> _userManager;
+
+    private DailyCheckPingService service;
+
+    public CheckIfAnyActiveInvitationsRequireAdminPing()
+    {
+      var store = new Mock<IUserStore<ApplicationUser>>();
+      _userManager = new Mock<UserManager<ApplicationUser>>(store.Object, null, null, null, null, null, null, null, null);
+      _emailService = new Mock<IEmailService>();
+      _repository = new Mock<IRepository<Invitation>>();
+      service = new DailyCheckPingService(_repository.Object, _emailService.Object, _userManager.Object);
+    }
+
+    [Fact]
+    public void ReturnsEmptyListGivenEmptyList()
+    {
+      var testlist = new List<Invitation>();
```

Comment:
This line appears 9 times in this file. It could probably be pulled up into a private field and initialized inline (or in the constructor if you prefer).